Sources/SWBCSupport/CMakeLists.txt

```diff
@@ -16,8 +16,15 @@ target_compile_definitions(SWBCSupport PRIVATE
   $<$<PLATFORM_ID:Windows>:_CRT_NONSTDC_NO_WARNINGS>)
 target_compile_options(SWBCSupport PRIVATE
   -fblocks)
+target_link_options(SWBCSupport PUBLIC
```

Comment:
I'm pretty sure this is the wrong way to ensure clients of this static library end up with a libc++ linkage

Reply:
Yeah, there isn't a great way to do this today. We need a mechanism to tell Swiftc to use clang++ as the clang-linker instead of clang, even when c++-interop isn't enabled. I've talked with @artemcm about it a bit, but will need something new in the driver. There are Swift runtime bits that the Swift driver knows about or I'd just say set the link language of the target to `CXX`, but that would break the Swift side.